transmissions/models.py

```diff
@@ -120,8 +122,9 @@ def send(self):
                 self.delete()
         except ChannelSendException:
             self.status = self.Status.FAILED
-        except:
+        except Exception as e:
             self.status = self.Status.BROKEN
+            self.exception = "{klass}: {message}".format(klass=e.__class__.__name__, message=str(e))
```

Comment:
Arguably, this could be done through a `logging` system rather than a permanent data store. I think `logging` would be a more suitable solution to introduce here since it's only assuming you've configured Django for it right now.

Either way you will want to make it configurable with settings. Logging is controlled by logging routes in settings, but if you introduce a new model/field, please add a setting.